backend/de.metas.acct.base/src/main/java/de/metas/acct/gljournal_sap/SAPGLJournalLine.java

```diff
@@ -88,4 +91,16 @@ public boolean isBaseTaxLine()
 		return parentId == null && taxId != null && !determineTaxBaseSAP;
 	}
 
+	@NonNull
+	public Optional<SAPGLJournalLine> getReverseChargeCounterPart(
```

Comment:
why not calling it "create" instead of "get"?

Reply:
hmm, fair enough